pkg/kubelet/apis/kubeletconfig/validation/validation.go

```diff
@@ -41,6 +42,60 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if kc.SystemCgroups != "" && kc.CgroupRoot == "" {
 		return fmt.Errorf("invalid configuration: system container was specified and cgroup root was not specified")
 	}
+	if utilvalidation.IsValidPortNum(int(kc.CAdvisorPort)) != nil {
+		return fmt.Errorf("Invalid configuration: --cadvisor-port %v must be between 1 and 65535", kc.CAdvisorPort)
```

Comment:
Please include the field name and the flag name in these error messages. The configuration can come from a set of flags or a file containing a serialized KubeletConfiguration. Eventually the flags will be removed, and then we'll remove the flag part of these error messages too. Also, please always specify inclusive or exclusive when describing ranges.

For now, this should do: `Invalid configuration: CAdvisorPort (--cadvisor-port) %v must be between 1 and 65535, inclusive`. And similarly for all the other errors.